lightllm/server/api_server.py

```diff
@@ -271,7 +271,7 @@ async def abort_request() -> None:
 
 def main():
     parser = argparse.ArgumentParser()
-    parser.add_argument("--host", type=str, default=None)
+    parser.add_argument("--host", type=str, default="127.0.0.1")
```

Comment:
127.0.0.1 won't work in ipv6, can you change it back? if you don't like `None` i think `0.0.0.0` may work as well